optuna/_hypervolume/utils.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
I believe this change is not relevant to this PR.
This change would be good to include when using features from ﻿`__future__`.

Reply:
Done!